docs/source/manual/jooq.rst

```diff
@@ -112,9 +112,85 @@ Below is an example of configuring the required binding uses a C3P0 pooling data
     }
   }
 
+Post commit actions
+~~~~~~~~~~~~~~~~~~~~
+
+Sometimes you need to perform an action only after the current transaction successfully commits. For example,
+updating or invalidating a cache.
+
+The JooqTransactionalHooks_ class provides the :code:`whenCommitted` method for registering callback actions that should be executed after a transaction is successfully committed. Any no-arguments lambda expression or :code:`Runnable` implementation can be passed as the action.
```

Comment:
nit. you have an `_` appended to on `JooqTransactionalHooks_`

Reply:
It's intentional: https://github.com/cerner/beadledom/pull/140#discussion_r297306196

Reply:
Sorry I missed the previous comment. The link has the trailing _ when I view the page in github, https://github.com/cerner/beadledom/blob/9fb82820a6b7e1c326a58907464a9fdacce4eb1f/docs/source/manual/jooq.rst while the other links don't.

Reply:
I could be mistaken, but I believe it doesn't render correctly since the file being linked to doesn't actually exist in master yet.